src/Common/src/System/Diagnostics/Debug.Unix.cs

```diff
@@ -8,108 +8,118 @@ namespace System.Diagnostics
 {
     // Intentionally excluding visibility so it defaults to internal except for
     // the one public version in System.Diagnostics.Debug which defines
-    // another version of this partial class with the public visibility 
+    // another version of this partial class with the public visibility
     static partial class Debug
     {
-        private static string NewLine => "\n"; 
+#if !MONO
```

Comment:
So with that we would still need the following both in `Debug.Unix.cs` and `Debug.Windows.cs`
```
static partial class Debug
{
#if MONO
static class Windows // for Debug.Windows.cs
{
#endif

#if MONO
    internal
#endif
    static string NewLine => "\n\r"; // for Debug.Windows.cs

    internal static IDebugLogger s_logger = new WindowsDebugLogger();

    internal sealed class WindowsDebugLogger : IDebugLogger
    {
        .......
    }
#if MONO
}
#endif
}
```

And that's because in case we are not in Mono, we still need a `Debug.NewLine` method and a `Debug.s_logger` field, and we cannot access it from `Windows.NewLine` or `Unix.NewLine` from `Debug.cs` because `PlatformHelper` is a Mono only class.